async-await/src/chat.rs

```diff
@@ -1,9 +1,6 @@
 #![feature(await_macro, async_await, futures_api)]
```

Comment:
The one thing I'd change is dropping the `futures_api` feature, as the latest nightly—2019-04-25—doesn't require this feature.

Reply:
Yes... I'm just trying to get CI to pass... which is turning out to suck.